pipeline/pipeline.go

```diff
@@ -17,25 +16,21 @@ import (
 	"github.com/streamingfast/substreams/manifest"
 	"github.com/streamingfast/substreams/orchestrator"
 	pbsubstreams "github.com/streamingfast/substreams/pb/sf/substreams/v1"
-	"github.com/streamingfast/substreams/pipeline/outputs"
 	"github.com/streamingfast/substreams/state"
 	"github.com/streamingfast/substreams/wasm"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/codes"
 	ttrace "go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
-	"google.golang.org/protobuf/proto"
-	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 type Pipeline struct {
 	vmType    string // wasm/rust-v1, native
 	blockType string
 
-	requestedStartBlockNum uint64 // rename to: requestStartBlock, SET UPON receipt of the request
-	maxStoreSyncRangeSize  uint64
-	isSubrequest           bool
+	//requestedStartBlockNum uint64 // rename to: requestStartBlock, SET UPON receipt of the request
+	maxStoreSyncRangeSize uint64
+	isSubrequest          bool
```

Comment:
why isn't `isSubrequest` part of the `requestContext` ? Isn't it context about the request?